server/src/main/java/io/crate/analyze/AnalyzedTableElements.java

```diff
@@ -89,7 +90,8 @@
      * additional primary keys that are not inline with a column definition
      */
     private List<T> additionalPrimaryKeys = new ArrayList<>();
-    private Map<T, Set<String>> copyToMap = new HashMap<>();
+
+    private Map<String, List<T>> ftSourcesMap = new HashMap<>();
```

Comment:
I think we used a `Set` before to avoid duplicates. Isn't this needed anymore?

Reply:
previously `Set` was holding dedicated indices names now they are keys of the `Map`.

on List: it holds column names and I thought we cannot declare same column twice in dedicated index declaration but after local testing looks like we can.

I will create a small separate bug fix PR to ensure that we cannot do smth like 
`dedicated_fulltext using fulltext(title, title)` - then would be fine to keep `List`?

Reply:
> I will create a small separate bug fix PR to ensure that we cannot do smth like
dedicated_index using fulltext(title, title) - then would be fine to keep List?

Sounds good.